openfisca_uk/reforms/basic_income/reform_1.py

```diff
@@ -59,63 +64,52 @@ def formula(person, period, parameters):
             person("age", period) < 65
         )
         return (
-            person("is_senior", period) * 50
-            + adult_young * 55
-            + adult_old * 65
-            + person("is_child", period) * 22
+            person("is_senior", period) * 40
```

Comment:
At some point it'd be good to have these as parameters, in the meantime could consider making them their own constants for more clarity, e.g. 
```
UBI_SENIOR = 40
...
return (
    person("is_senior", period) * UBI_SENIOR
...
```